WcaOnRails/env_config.rb

```diff
@@ -30,7 +30,7 @@
     optional :SIDEKIQ_REDIS_URL, :string, ''
     optional :DISCOURSE_URL, :string, ''
     optional :STORAGE_AWS_BUCKET, :string, ''
-    optional :STORAGE_AWS_REGION, :string, ''
+    optional :STORAGE_AWS_REGION, :string, 'us-west-2'
```

Comment:
Not a big fan of hardcoding this here.
Might cause a lot of scratching heads "Why am I automatically being thrown to `us-west-2` without even specifying anything?" down the line.
If you really need this default value, change it in the (checked-in!) `.env.development` file.

Reply:
makes sense! Moved it